sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -53,28 +80,39 @@ typedef struct
   az_span option;
 } az_storage_blobs_blob_upload_options;
 
+/**
+ * @brief Init a client with default options
+ * This is convenient method to create a client with basic settings
+ * Specific options can be modified after initializing the options
+ *
+ * @remark Use this, for instance, when only caring about setting one option by calling this method
+ * and then overriding that specific option
+ */
 AZ_NODISCARD az_storage_blobs_blob_client_options az_storage_blobs_blob_client_options_default();
 
+/**
+ * @brief Init a blob upload with default options
+ * This is convenient method to create a blob upload options with basic settings
+ * Specific options can be modified after initializing the options
+ *
+ */
 AZ_NODISCARD AZ_INLINE az_storage_blobs_blob_upload_options
 az_storage_blobs_blob_upload_options_default()
 {
   return (az_storage_blobs_blob_upload_options){ .option = AZ_SPAN_NULL };
 }
 
-typedef struct
-{
-  az_span option;
-} az_storage_blobs_blob_download_options;
-
 /**
- * @brief Creates a new blob
+ * @brief Uploads to a blob
  *
  * @param client a storage blobs client structure
+ * @param context supports cancelling long running operations
  * @param content blob content
- * @param options create options for blob. It can be NULL so nothing is added to http request
- * headers
+ * @param options upload options for blob. 
```

Comment:
```suggestion
 * @param options The upload options for blob. 
```

Reply:
Hmm... interesting. We don't use the options at all, currently.

Reply:
The options exists for future proofing.  We don't need it now,  We can't change the signature in the future when we may need them.

Reply:
Makes sense.